arangosh/Shell/V8ShellFeature.cpp

```diff
@@ -511,6 +511,31 @@ int V8ShellFeature::runShell(std::vector<std::string> const& positionals) {
       input = "help()";
     }
 
+    if (i[0] == 'a') {
+      auto pos = i.find(' ');
+      if (pos != std::string::npos) {
+        i = i.substr(0, pos);
+      }
+      if (i == "arangobackup" ||
+          i == "arangobench" ||
+          i == "arangod" ||
+          i == "arangodbtests" ||
+          i == "arangodump" ||
+          i == "arangoexport" ||
+          i == "arangoimp" ||
+          i == "arangoimport" ||
```

Comment:
This will catch `arangoimport ` (with trailing whitespace`, but not `arangoimport` (without).
In addition, it will catch things like 
```
arangoimport = db.collection.insert(docs);
```
which will now suddenly stop working.